code/__DEFINES/food.dm

```diff
@@ -11,6 +11,7 @@
 #define GROSS 		(1<<10)
 #define TOXIC 		(1<<11)
 #define PINEAPPLE	(1<<12)
+#define CLOTH (1<<14)
```

Comment:
Why 14th byte and not 13th? Also, UNALIGNED REEE.

Reply:
I feel the foodtype PR is gonna get merged and this basically makes it compatible with that without having to change anything because no double 13.
That is either clever of me or really dumb as fuck.

Reply:
you are gonna get merge conflict'd if it merges anyway so there is no point

Reply:
Please don't do this in the future.